Daan is erbij

gbxdl opened the pull request:
# Gezellig

gbxdl committed:
Daan is erbij

Marruk commented on src/bots/jonas.ts:
```diff
@@ -5,13 +5,15 @@
 // - je meer geld hebt dan het laatste hoogste bod
 // - je niet het hoogste bod hebt
 // komt eigenlijk gewoon neer als het nog zin heeft om een bod te doen dus ja
+import type {PlayerKey} from "@/models/auction.models.ts";
```
Ja dit leek me onhandig voor nu omdat het dan niet met alleen dit bestand zichtbaar is wat je allemaal door kan krijgen snap je

gbxdl replied:
oja ok

Marruk commented on src/bots/daan.js:
Wil je geen Typescript? Kan ook he, minder bugs

gbxdl replied:
ok ja slim

gbxdl committed:
ok nu alleen nog de comments verder is ie perfect.

Marruk commented on src/state/auction.state.ts:
```diff
@@ -16,7 +16,7 @@ export const initialState: State = {
     key: bot.key,
     bot,
-    moneyLeft: 10000000,
+    moneyLeft: 10_000_000,
```
Wat is dit voor vage vibe coding troep?

gbxdl replied:
haha

Marruk replied:
Wow ik wist niet dat dit kon

Marruk commented on src/bots/jonas.ts:
```diff
@@ -5,15 +5,14 @@
   riderBib: number, // nummer zoals op https://www.procyclingstats.com/race/vuelta-a-espana/2025/startlist/alphabetical (-1 als het niet bekend is)
   highestBid: number | null, // hoogste bod, is nooit van jou
-  highestBidBy: PlayerKey | null, // hoogste bod persoon
+  highestBidBy: 'tadej' | 'jonas' | null, // hoogste bod persoon
```
'daan' moet hier wel bij he

gbxdl replied:
dan moet het wel gewoon playerkey zijn dan toch, anders zit iedereen Jonas aan te passen

Marruk replied:
Ja maar dat is fijn want dan kan je jonas.ts gebruiken als je een endpoint wil bouwen ofzo

Marruk commented on src/bots/daan.ts:
```diff
@@ -0,0 +1,32 @@
+  comment: string | null
+}
+
+export function bot(
```
Je kan jonas als basis nemen hier, dan heb je alle interfaces niet nodig

Marruk replied:
Nou ja maakt ook niet zoveel uit

gbxdl committed:
okok nu echt finalfinalV2final

Marruk commented on src/bots/jonas.ts:
```diff
@@ -10,7 +10,7 @@ export function bot(
-  highestBidBy: 'tadej' | 'jonas' | null, // hoogste bod persoon
+  highestBidBy: 'tadej' | 'jonas' | 'daan' | null, // hoogste bod persoon
   bids: { // alle boden (in oplopende volgorde), inclusief die van jou
     player: 'tadej' | 'jonas', // naam
```
Lol hier ook nog

Marruk replied:
En op regel 28

gbxdl replied:
ja daarom playerkey snap je

Marruk replied:
Ja maar ja

gbxdl committed:
finalV2.5 final
maar ik zou gewoon playerkey in jonas doen hoor anders zit je dit steeds te doen.
